grammars/css.cson

```diff
@@ -516,7 +516,7 @@
         'captures':
           '1':
             'name': 'punctuation.definition.entity.css'
-        'match': '(:+)(after|before|content|first-letter|first-line|host|(-(moz|webkit|ms)-)?selection)\\b'
+        'match': '(::?)(after|before|content|first-letter|first-line|host|(-(moz|webkit|ms)-)?selection)\\b'
```

Comment:
👍  @idbartosz

"`after`, `before`, … are prefixed by …" -> "`after`, `before`, … can be prefixed by …"